src/test/app/forms/validation/validators/amountIsDefinedConstraint.ts

```diff
@@ -0,0 +1,134 @@
+import { expect } from 'chai'
+import { ValidationArguments } from 'class-validator'
+import { AmountIsDefinedConstraint } from 'app/forms/validation/validators/amountIsDefined'
+
+describe('AmountIsDefinedConstraint', () => {
+  const constraint: AmountIsDefinedConstraint = new AmountIsDefinedConstraint()
+
+  describe('validate', () => {
+    describe('should return true', () => {
+      it('given a value is null', () => {
+        const args: ValidationArguments = {
+          'value': '',
```

Comment:
key of object doesn't have to be wrapped by ' ' as long as it's valid name of variable.